fixtures/redshift_schema_edl.sql

```diff
@@ -0,0 +1,241 @@
+CREATE SCHEMA IF NOT EXISTS {redshift_schema_edl};
+
+--
+-- TODO: EDL cutover?
+--
+-- CREATE TABLE IF NOT EXISTS {redshift_schema_edl}.canvas_api_enrollments
+-- (
+--     course_id VARCHAR NOT NULL,
+--     user_id VARCHAR NOT NULL,
+--     term_id VARCHAR(4) NOT NULL,
+--     last_activity_at TIMESTAMP,
+--     feed TEXT NOT NULL
+-- );
+
+CREATE TABLE IF NOT EXISTS {redshift_schema_edl}.student_degree_progress
+(
+    sid VARCHAR NOT NULL,
+    feed TEXT NOT NULL
+);
+
+--
+-- TODO: EDL cutover?
+--
+-- CREATE TABLE IF NOT EXISTS {redshift_schema_edl}.sis_api_profiles
+-- (
+--     sid VARCHAR NOT NULL,
+--     feed TEXT NOT NULL
+-- );
+
+--
+-- TODO: EDL cutover?
+--
+-- CREATE TABLE IF NOT EXISTS {redshift_schema_edl}.sis_api_profiles_v1
+-- (
+--     sid VARCHAR NOT NULL,
+--     feed TEXT NOT NULL
+-- );
```

Comment:
@johncrossman The v1 sis_api_profiles currently exists as a fallback to an older Hub interface. To the extent we need to replicate anything on the Hub SIS API side, only the newer sis_api_profiles should be necessary (assuming we actually want to rebuild the entire rather complicated SIS API data structure).